v2/cli/term/log_page.go

```diff
@@ -28,8 +31,11 @@ func newLogPage(
 		logText: tview.NewTextView().SetDynamicColors(true),
 	}
 
+	// GitHub places a restriction on the text field for a Check Run at 65535
+	// characters, so we use this as a maximum and truncate if needed.
+	l.maxBytes = 65535
 	l.logText.SetBorder(true).SetTitle("Logs (<-/Del) Quit")
-	l.logString = "Waiting for logs..."
+	l.logBuf, _ = circbuf.NewBuffer(l.maxBytes)
```

Comment:
Don't ignore errors...

Reply:
How should I deal with errors inside of `newLogPage()`? Should I just return an empty page with an error message, since the buffer couldn't be initialized?

Reply:
I think there may be two reasonable ways to deal with errors.

Yes... one of them would be development of an error page, but I'd about the complexity found therein creating the possibility that an initial error is obscured by an error in the error page itself.

You know tview better than I do... is there a way to just shut the UI down on command and then display an error to stderr like normal?

Reply:
I should add that centralizing the error handling for term probably needs to be its own separate PR.

To account for my "don't ignore errors" comment, for now, just add a comment with a TODO.

Reply:
Sure. I made a change to initialize the buffer inside the log page's load function instead, so if there is an error, I can display that to the floating log box that comes up and return before starting the two goroutines for streaming logs. 

This would make displaying the error message possible, since I can create the buffer after creating the textview. I have a feeling that this would be a better user experience than shutting down the entire UI. What do you think?

Reply:
It works well for this case. It doesn't work well for the general case. It's fine for now, but longer term, we need something else.

Reply:
Yeah I was thinking the same. We can add a standardized system for error handling in the future.